use_cases/init-process-for-fluent-bit/README.md

```diff
@@ -138,6 +138,40 @@ You can use them as env vars directly in the Fluent Bit config.
     region 		${AWS_REGION}
 ```
 
+### How to ignore the generated FireLens configuration?
+
+Please read the [Fluent Bit Out of Memory Kill Prevention Guide](https://github.com/aws-samples/amazon-ecs-firelens-examples/tree/mainline/examples/fluent-bit/oomkill-prevention#full-firelens-configuration-examples). It discusses settings to limit the memory usage of the Fluent Bit process and enable file buffering. These settings are specified on the input definition; in [FireLens the input configuration that collects stdout logs is autogenerated](https://aws.amazon.com/blogs/containers/under-the-hood-firelens-for-amazon-ecs-tasks/) by ECS. Therefore, enabling buffer control settings requires [overriding the generated FireLens input configuration](https://aws.amazon.com/blogs/containers/how-to-set-fluentd-and-fluent-bit-input-parameters-in-firelens/). 
+
+The init process can help with this. Set the environment variable `aws_fluent_bit_init_ignore_firelens_config` on the container definition for the init image:
+
+```
+            "environment": [
+                {
+                    "name": "aws_fluent_bit_init_ignore_firelens_config",
+                    "value": "true"
+                },
+            ],
+```
+
+And then make sure one of the configuration files specified for the init process contains an input definition like the following to collect stdout logs:
+
+```
+[INPUT]
+     Name forward
+     unix_path /var/run/fluent.sock
+     Mem_Buf_Limit 100MB
+```
```

Comment:
It may be helpful to add an example with the appropriate match statement for docker logs.
```
[OUTPUT]
    Name                cloudwatch_logs
    Match               Application-firelens-*
```